python/cudf/cudf/_libxx/quantiles.pyx

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+import pandas as pd
+
+from cudf._libxx.lib cimport *
+from cudf._libxx.column cimport Column
+from cudf._libxx.table cimport Table
+cimport cudf._libxx.includes.quantiles as cpp_quantiles
+
+
+def quantiles(Table source_table, q, interp, is_input_sorted, column_order, null_precedence):
```

Comment:
Would Cython type checking be useful for any of the other parameters here?

Reply:
I was able to add `vector[double] q`, but the others extend `Enum` which is not an extension type. :(

Reply:
Even on Python based parameters you can mark them as `object` so that Cython can be smarter about how it handles them. It also helps us as developers 😄